docs/Developer-Guide.md

```diff
@@ -304,7 +304,7 @@ You MUST choose one of `alpine`, `centos`, `clearlinux`, `debian`, `euleros`, `f
 > - You should only do this step if you are testing with the latest version of the agent.
 
 ```
-$ sudo install -o root -g root -m 0550 -t ${ROOTFS_DIR}/bin ../../../src/agent/target/x86_64-unknown-linux-musl/release/kata-agent
+$ sudo install -o root -g root -m 0550 -t ${ROOTFS_DIR}/usr/bin ../../../src/agent/target/x86_64-unknown-linux-musl/release/kata-agent
```

Comment:
@stevenhorsman, 

Let me write down here the full path I took to check your PR and, please, correct me if this is different from what you did.

Let's take a look at the [agent service](
https://github.com/kata-containers/kata-containers/blob/0c463babf3c6ef3645fbd0f1f5adfc9541fc74bf/src/agent/kata-agent.service.in):
```
#
# Copyright (c) 2018-2019 Intel Corporation
#
# SPDX-License-Identifier: Apache-2.0
#

[Unit]
Description=Kata Containers Agent
Documentation=https://github.com/kata-containers/kata-containers
Wants=kata-containers.target

[Service]
# Send agent output to tty to allow capture debug logs
# from a VM vsock port
StandardOutput=tty
Type=simple
ExecStart=@BINDIR@/@AGENT_NAME@
LimitNOFILE=1048576
# ExecStop is required for static agent tracing; in all other scenarios
# the runtime handles shutting down the VM.
ExecStop=/bin/sync ; /usr/bin/systemctl --force poweroff
FailureAction=poweroff
# Discourage OOM-killer from touching the agent
OOMScoreAdjust=-997
```

The part that's interesting for us is:
```
ExecStart=@BINDIR@/@AGENT_NAME@
```

So, it depends on whatever as set as BINDIR.  BINDIR is actually set [here](https://github.com/kata-containers/kata-containers/blob/0c463babf3c6ef3645fbd0f1f5adfc9541fc74bf/src/agent/Makefile#L70) and is set to:
```
BINDIR := /usr/bin
```

So, that's the reason I truly believe this patch should get in.

Reply:
Thanks - it's always good to see the original mechanics that match the doc :)